### PR TITLE
Add New props type tabBarActiveStyle(Object) to Style Active tab

### PR DESCRIPTION
**Pull request by dinesh829269**
Styling Active tab  like giving background color, etc is difficult so i put a props tabBarActiveStyle for user to define activetab style

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -273,6 +273,9 @@ const ScrollableTabView = React.createClass({
     if (this.props.tabBarUnderlineStyle) {
       tabBarProps.underlineStyle = this.props.tabBarUnderlineStyle;
     }
+    if (this.props.tabBarActiveStyle) {
```
Comment (skv-headless) [inline index.js]: I prefer not to pass such props throw index.js. Just keep in tab bar.

File: DefaultTabBar.js [hunk 2]
```diff
@@ -27,17 +27,19 @@ const DefaultTabBar = React.createClass({
       activeTextColor: 'navy',
       inactiveTextColor: 'black',
       backgroundColor: null,
+      activetabStyle: null,
```
Comment (dinesh829269) [inline DefaultTabBar.js]: There are changes in the renderTab function also

**Comment by skv-headless** [comment]
Please squash commits to one.

**Comment by dinesh829269** [comment]
There was no Active tab bar style , i was needed this type of tabbar , so i did the changes
simulator screen shot 05-oct-2016 6 07 28 pm

File: DefaultTabBar.js (hunk 2 above)
Comment (skv-headless) [inline DefaultTabBar.js]: please rename to ```activeTabStyle```

**Comment by skv-headless** [comment]
Can you please make 1 pr?

**Comment by amyliu1062** [comment]
why not to update module with this all ?

**Comment by amyliu1062** [comment]
when the code will  merge ?

**Comment by amyliu1062** [comment]
@skv-headless   how can i change the backgroundColor of active tab ?

**Comment by amyliu1062** [comment]
@skv-headless  i want to realize a demo that the tab item has a title and a icon . The following rendering
the facebook demo has a icon by use 'tabLabel' 
how i get a title ?  

image

**Comment by skv-headless** [comment]
https://github.com/skv-headless/react-native-scrollable-tab-view#injecting-a-custom-tab-bar
then do what you want with tab bar.

**Comment by amyliu1062** [comment]
@skv-headless  i need title and icon at the same time . but view only one tabLabel . like this:
```
     <ScrollableTabView>
        <ReactPage tabLabel="React" />
        <FlowPage tabLabel="Flow" />
        <JestPage tabLabel="Jest" />
      </ScrollableTabView>
```
image

or like this:
```
   <ScrollableTabView
            initialPage={1}
            renderTabBar={() => <FacebookTabBar />}
        >
            <ScrollView tabLabel="ios-paper" style={styles.tabView}>
                <View style={styles.card}>
                    <Text>News</Text>
                </View>
            </ScrollView>
            <ScrollView tabLabel="ios-people" style={styles.tabView}>
                <View style={styles.card}>
                    <Text>Friends</Text>
                </View>
            </ScrollView>
            <ScrollView tabLabel="ios-chatboxes" style={styles.tabView}>
                <View style={styles.card}>
                    <Text>Messenger</Text>
                </View>
            </ScrollView>
            <ScrollView tabLabel="ios-notifications" style={styles.tabView}>
                <View style={styles.card}>
                    <Text>Notifications</Text>
                </View>
            </ScrollView>
            <ScrollView tabLabel="ios-list" style={styles.tabView}>
                <View style={styles.card}>
                    <Text>Other nav</Text>
                </View>
            </ScrollView>
        </ScrollableTabView>;
```
image
----
 
so i copy Facebook Component , and add a `Text` Componnet like this:
```
render() {
        return <View style={[styles.tabs, this.props.style, ]}>
            {this.props.tabs.map((tab, i) => {
                return <TouchableOpacity key={tab} onPress={() => this.props.goToPage(i)} style={styles.tab}>
                    <Icon
                        name={tab}
                        size={30}
                        color={this.props.activeTab === i ? 'rgb(59,89,152)' : 'rgb(204,204,204)'}
                        ref={(icon) => { this.tabIcons[i] = icon; }}
                    />
                    <Text
                        color={this.props.activeTab === i ? 'rgb(59,89,152)' : 'rgb(204,204,204)'}
                        ref={(name) => { this.names[i] = name; }}>name</Text>
                </TouchableOpacity>;
            })}
        </View>;
    },
```
but i don`t know  how get the text . can you give me a message ?
image .

**Comment by srhtylmz19** [comment]
@dinesh829269 can you please show me how to change active background color?

**Pull request closed without merging**